core/object/object.cpp

```diff
@@ -1163,10 +1163,31 @@ Error Object::emit_signalp(const StringName &p_name, const Variant **p_args, int
 		uint32_t idx = 0;
```

Comment:
Just as a note, we should replace this LocalVector<Connection> by an alloca() for performance, so we use the stack for this and not allocate memory every single time signals are emitted, which is very inefficient.